src/fhir_populator/populator.py

```diff
@@ -424,9 +442,79 @@ def get_latest_package_version(self, package_name: str) -> str:
         self.log.debug(f"Latest version: {last_version}")
         return last_version
 
+    def persistate(self, dependency_graph):
+        def copy_directory_structure(src, dst):
+            """
+            Recursively creates directories from src to dst
+            """
+            for root, dirs, files in os.walk(src):
```

Comment:
nested for loops are not necessary, `os.makedirs` is already recursive (like `mkdir -p`)